examples/stm32f4/src/bin/adc.rs

```diff
@@ -24,19 +24,40 @@ async fn main(_spawner: Spawner) {
     // Startup delay can be combined to the maximum of either
     delay.delay_us(Temperature::start_time_us().max(VrefInt::start_time_us()));
 
+    let vref_sample = adc.read_internal(&mut vrefint);
+
+    let convert_to_millivolts = |sample| {
+        // From http://www.st.com/resource/en/datasheet/DM00071990.pdf
+        // 6.3.24 Reference voltage
+        const VREF_MILLIVOLTS: u32 = 1210; // mV
```

Comment:
Could you rename this to `VREFINT_MILLIVOLTS` and other variables that refer to `VrefInt`? Also in other examples.

So that it's not confused with actual `Vref`, which is an external pin (on smaller chips connected internally to VDDA).